### PR TITLE
[corda-ent] Added Ansible Role and tpl file for automation of gateway service.

### PR DESCRIPTION
**Pull request by nikhilagolla**
Signed-off-by: nikhilagolla <nikhila.golla@accenture.com>

**Changelog**
- Add Ansible Role and tpl file for automation of gateway service.
 

**Reviewed by**
@sownak @jagpreetsinghsasan @suvajit-sarkar 

 

**Linked issue**
#1734

### CODE REVIEW

File: platforms/r3-corda-ent/configuration/roles/helm_component/templates/gateway.tpl [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+      replicas: 1
+    service:
+      type: ClusterIP
+      port: 8080
+    zoneName: {{ org.services.zone.name }}
+    zonePort: 12345
+    gatewayPort: {{ org.services.gateway.port }}
+    authName: {{ org.services.auth.name }}
+    authPort: {{ org.services.auth.port }}
+
```
Comment (suvajit-sarkar) [inline platforms/r3-corda-ent/configuration/roles/helm_component/templates/gateway.tpl:62]: new line

File: platforms/shared/configuration/roles/helm_lint/vars/main.yaml [hunk 2]
```diff
@@ -59,3 +60,4 @@ charts:
   vault_kubernetes_job: vault_kubernetes
   crypto_ibft_job: crypto_ibft
   operations_console: operations_console
+
```
Comment (suvajit-sarkar) [inline platforms/shared/configuration/roles/helm_lint/vars/main.yaml:63]: new line

Comment (nikhilagolla) [reply platforms/shared/configuration/roles/helm_lint/vars/main.yaml]: fixed

File: platforms/r3-corda-ent/configuration/roles/helm_component/templates/gateway.tpl (hunk 1 above)
Comment (nikhilagolla) [reply platforms/r3-corda-ent/configuration/roles/helm_component/templates/gateway.tpl]: Fixed

**Review by suvajit-sarkar — APPROVED** [review]

**Pull request merged**